fred/cogs/dialogflow.py

```diff
@@ -1,32 +1,32 @@
 import asyncio
 import json
-import logging
-import os
 import uuid
+from os import getenv
 
 import nextcord
-from nextcord.ext import commands
 from google.cloud import dialogflow
 from google.oauth2 import service_account
 
-import config
-from libraries import common
+from .. import config
+from ..libraries import common
 
-
-if os.environ.get('DIALOGFLOW_AUTH'):
-    DIALOGFLOW_AUTH = json.loads(os.environ.get("DIALOGFLOW_AUTH"))
+if env_auth_str := getenv("DIALOGFLOW_AUTH"):
+    DIALOGFLOW_AUTH = json.loads(env_auth_str)
     session_client = dialogflow.SessionsClient(
         credentials=service_account.Credentials.from_service_account_info(DIALOGFLOW_AUTH)
     )
     DIALOGFLOW_PROJECT_ID = DIALOGFLOW_AUTH["project_id"]
     SESSION_LIFETIME = 10 * 60  # 10 minutes to avoid repeated false positives
+else:
+    DIALOGFLOW_AUTH = None
+    session_client = None
+    DIALOGFLOW_PROJECT_ID = None
 
 
-class DialogFlow(commands.Cog):
-    def __init__(self, bot):
-        self.bot = bot
+class DialogFlow(common.FredCog):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
         self.session_ids = {}
-        self.logger = logging.Logger("DIALOGFLOW")
```

Comment:
Why remove this ?

Reply:
Dialogflow now inherits from common.FredCog which has a logger that actually outputs (logging is weird). This has been applied to every Cog.